design.md

```diff
@@ -2,12 +2,26 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+This design is targeted towards someone with enough programming experience to write simple functions. The goal of the design is to provide more familiar syntax to a programmer. It also allows the user to be more flexible in how they evaluate the current surroundings of Picobot. 
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+We thought that the `*` and `x` states were too confusing and that it was too tedious to enforce directional order when describing the surroundings of the bot. Thus, we decided to let users only express the surrounding states relevant to desired behavior without constraining the directional order. We also decided that being able to express multiple commands at once would be more convenient than the current Picobot's behavior of separating every command into a new state.
+
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
 
+The `x` direction is more intuitive in this design, especially to someone with a programming background who is more accustomed to boolean values. The `*` behavior is also easier to express because it is the default state of a direction when the user does not specify anything for that direction. Recursive behavior within a state is also easier to express in our design. We designed this functionality in a `while` command so that the user does not need to think about the problem in terms of repeatedly calling a certain state. Ideally this behavior is more semantically intuitive to the user. 
```

Comment:
I'm not sure I see what's recursive, exactly. I guess I agree that the sort-of imperative approach your syntax has probably would make more sense to a lot of programmers for picobot than the original picobot's more declarative approach does, though.